types/v56/types.go

```diff
@@ -2435,3 +2437,27 @@ type PortGroupRecordType struct {
 	ScopeType     int     `xml:"scopeType,attr,omitempty"` // Scope of network using the portgroup(1=Global, 2=Organization, 3=vApp)
 	Link          []*Link `xml:"Link,omitempty"`
 }
+
+// Represents org VDC Network
+// Reference: vCloud API 27.0 - Org VDC Network
+// https://code.vmware.com/apis/72/doc/doc/types/QueryResultOrgVdcNetworkRecordType.html
+type QueryResultOrgVdcNetworkRecordType struct {
```

Comment:
Just curious, how come this struct was not needed before and is needed now?

Reply:
Left as was doing other POC. This just for future use